testing/describes.js

```diff
@@ -445,10 +445,13 @@ class IntegrationFixture {
 
   /** @override */
   setup(env) {
-    const body = this.spec.body;
+    const body = typeof this.spec.body == 'function' ?
+          this.spec.body() : this.spec.body;
+    const experiments = this.spec.experiments || [''];
```

Comment:
I was thinking people may not pass in experiments, and causes experiments undefined. Let me find a more elegant way to check this